recipes/perses/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "perses" %}
+{% set version = "0.8.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/choderalab/perses/archive/{{ version }}.tar.gz
+  sha256: adafc9aac88cba3a2a103a814520aa1c58728513f784cd71d7fdca9cdaa3dc0d
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps"
+  entry_points:
+  - perses-relative = perses.app.setup_relative_calculation:run
+  - perses-fah = perses.app.fah_generator:run
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - numpy
+  run:
+    - python >=3.6
+    - numpy
+    - scipy
+    - numba
+    - netcdf4
+    - matplotlib-base
+    - seaborn
+    - lxml
+    - networkx >=2.0
+    - dask
+    - distributed
+    - progressbar2
+    - tqdm
+    - dask-jobqueue
+    - dicttoxml
+    - fire
+    - joblib
+    - mdtraj
+    - pymbar
+    - ambertools >=19.11
+    - openmm >=7.4.2
+    - openmmforcefields
+    - openmoltools >=0.8.4
+    - openmmtools >=0.20.0
+    - yank
+    - pdbfixer
+    # - openeye-toolkits
```

Comment:
Is there a reason why this isn't being added to conda-forge? (point me to a comment if there already is one)

Reply:
Licensing issues, sadly They are only available through the [company's channel](https://anaconda.org/openeye/openeye-toolkits). 

We work around that requirement using try-guarded imports for that package, and make sure to mention the extra package in the errors plus documentation.